server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/application/Application.java

```diff
@@ -220,4 +276,31 @@ private boolean isValidApplicationSuffix(String suffix) {
     public String getPathToApplication() {
         return pathToApplication;
     }
+
+    /**
+     true if the application is a real application
+     false if the application is a dummy application e.g. a service
```

Comment:
I dont like the name dummy application. It reminds me of testing. 

Calling a Application `Application` and a Service `Service` sounds way better.

Is there a case in which a "Dummy Application" is not a service. In that case my proposed naming is of cource not that good